rstudio/rstudio
# Theme update to User and News

### Intent

> Describe briefly what problem this pull request resolves, or what new feature it introduces. Include screenshots of any new or altered UI. Link to any Github issues that are related. 

Related to issue https://github.com/rstudio/rstudio-pro/issues/6373
Pro PR: https://github.com/rstudio/rstudio-pro/pull/6375


Updates theme for user and news.

New theme has things like dark mode, a new footer icon, accessibility fixes, etc.

I also fixed the toc depth / expansion w/ scrolling for the news as well



### Approach

> Describe the approach taken and the tradeoffs involved if non-obvious; add an overview of the solution if it's complicated.

### Automated Tests

> Indicate whether this change includes unit tests or integration tests, or link a work item or pull request to add those tests to another repo. If the change cannot or will not be covered by a test, indicate why.

### QA Notes

> Add additional information for QA on how to validate the change, paying special attention to the level of risk, adjacent areas that could be affected by the change, and any important contextual information not present in the linked issues. 

- [x] IDE User
- [x] News

Verify that the new posit-docs theme version is installed:

```
title: posit-docs
author: Ashley Henry, David Aja, Aron Atkins
version: 4.0.2
quarto-required: ">=1.3.340"
```

Spot-check items in UI:

-  Introduction of dark mode (the easiest thing to spot)
- New footer posit icon: 
<img width="146" alt="image" src="https://github.com/rstudio/rstudio-pro/assets/31460023/69e44f14-06f4-4766-9e75-400edc11896c">
 - Search box (older version in dark mode was green, teal, etc. - it now uses blues and gray)
<img width="682" alt="2024-05-28_18-13-59" src="https://github.com/rstudio/rstudio-pro/assets/31460023/27112b2a-ebca-410d-bc75-282182939d2c">
- GA file and configuration is included in the theme now.

### Documentation
> Specify which documentation has been added or modified and why (User Guide? Admin Guide?). If no documentation was added for a new feature, indicate why. 

- IDE User
- News

### Checklist
